icubam/backoffice/templates/icu.html

```diff
@@ -70,14 +70,24 @@ <h3 class="card-title">
               </div>
               <div class="col-25">
                 <label for="city">{{ _('City') }}</label>
-                <input type="text" id="city" name="city" class="form-control" placeholder="{{ _('City') }}"
+                <input
+                  name="city" list="city_options"
+                  class="form-control"
+                  placeholder="{{ _('City') }}"
                   value="{% if icu.city %}{{icu.city}}{% end %}" required autofocus>
+                </input>
+                <datalist id="city_options"></datalist>
               </div>
 
-              <div class="col-25">
-                <label for="dept">{{ _('Department / State') }}</label>
-                <input type="text" id="dept" name="dept" class="form-control" placeholder="{{ _('Department') }}"
+              <div class="col-25 autocomplete">
```

Comment:
Does the `autocomplete` class do anything? Just curious.

Reply:
no you are absolutely write. I started with another way and forgot to remove it. good catch. it is now removed.